packages/wrangler/src/deployment-bundle/entry.ts

```diff
@@ -59,7 +59,7 @@ export async function getEntry(
 			args.legacyAssets ||
 			config.legacy_assets ||
 			args.experimentalAssets ||
-			config.experimental_assets?.some((x) => x.directory)
+			config.experimental_assets?.directory
```

Comment:
Should this be just `config.experimental_assets`? Once you have this object the `directory` cannot be undefined (and I think we should have validation that it cannot be empty).

Reply:
true. your comment however triggered smth else tho, and made me wonder what happens if `directory` is an empty string....and sure enough, `directory = ""` passes validation just fine. To be clear, this seems to be a general thing, not the `experimental_assets` in particular. Feels like we should handle this use case one way or the other, either by failing validation, or resolving to `cwd`? Would that even be expected behaviour...the resolving to `cwd` I mean

Reply:
addressed here: https://github.com/cloudflare/workers-sdk/pull/6346